ksqldb-parser/src/test/java/io/confluent/ksql/util/QueryMaskTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright 2021 Confluent Inc.
+ *
+ * Licensed under the Confluent Community License (the "License"); you may not use
+ * this file except in compliance with the License.  You may obtain a copy of the
+ * License at
+ *
+ * http://www.confluent.io/confluent-community-license
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OF ANY KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package io.confluent.ksql.util;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import org.junit.Test;
+
+public class QueryMaskTest {
```

Comment:
Can we also add test coverage for requests which consist of multiple statements (including tests with a mix of valid and invalid statements)?

Also a test for `IF NOT EXISTS`, and an unquoted connector name.